### ISSUE TITLE
sway support?

### ISSUE THREAD

**Issue by AmarOk1412** [foss-project/green-recorder#82]
Hi!

I tried to used green recorder on gnome and sway, nice work!

But yeap, it didn't work on sway. So i'm wondering which wm do you want to support?

Have a nice day!

**Comment by HaoZeke** [comment]
I too would love to see this implemented with the new `wlroots` backend.

**Comment by SleepyMario** [comment]
+1

**Comment by zajca** [comment]
I tried green recorder on sway, app is starting and recording but output is black with pointer moving :/

**Comment by AmarOk1412** [comment]
@zajca yes, it's the same problem on all apps trying to record a Wayland session unless you use some ugly tricks.

**Comment by Jpfonseca** [comment]
On what @AmarOk1412  first comment:

I just installed it and since I don't have any Gnome packages installed `green-recorder` won't work. The current version of `green-recorder` depends on Gnome Shell Screencast.

**Comment by keithbowes** [comment]
Yeah, it was disappointing that this claims to work in Wayland, but only works in one bloated, yet surprisingly lacking in features, compositor (works great in X, though, without the audio/video synchronization issues and extreme memory usage of recordmydesktop).  There's [wf-recorder](https://github.com/ammen99/wf-recorder), made for the Wayfire compositor but will work on anything with wlroots, but it seems to use patented audio and video formats not legal to use in my country.  There's also [wlstream](https://github.com/colemickens/wlstream), which uses Pulseaudio directly, so it shouldn't have the legal issues of wf-recorder, but I can't get it to work at all.

**Comment by Jpfonseca** [comment]
I think wf-recorder supports multiple formats. You can specify the codec... Worst case make a gif ;) .
I am also interested in seeing this package with full Wayland support. For now I am using wf-recorder.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by keithbowes** [comment]
I tried wf-recorder with libvpx.  It recorded the video, with very low quality.  I can't get audio to work at all, though.

**Comment by Jpfonseca** [comment]
Open a issue on wf-recorder project
________________________________
